spec/jobs/webflow_sync/update_item_job_spec.rb

```diff
@@ -5,9 +5,16 @@
 module WebflowSync
   RSpec.describe WebflowSync::UpdateItemJob, type: :job do
     let(:collection_slug) { 'articles' }
-    let(:article) { create(:article, webflow_item_id: 'webflow_item_id') }
+    let(:article) { create(:article, webflow_item_id: '60defde681813e53c6be97ea') }
     let(:mock_webflow_api) { instance_double('mock_webflow_api', update_item: nil) }
 
+    before(:each) do
+      WebflowSync.configure do |config|
+        config.webflow_site_id = 'webflow_site_id'
+        config.api_token = 'webflow_api_token'
```

Comment:
If we'd get these using `ENV.fetch`, then we wouldn't have to specify them like this.

Reply:
Thank you! Installed `dotenv-rails`, and start using` ENV.fetch`.